net/remote/remote-node.sh

```diff
@@ -129,6 +129,8 @@ cat >> ~/solana/on-reboot <<EOF
   # shellcheck source=/dev/null
   SUDO_OK=1 source scripts/tune-system.sh
 
+  sudo solana-sys-tuner &
+
```

Comment:
Please capture the pid into a .pid file so `solana-sys-tuner` gets killed on a `./net.sh stop`

Reply:
Done